package.json

```diff
@@ -62,6 +62,7 @@
     "node": ">=10.10.0"
   },
   "dependencies": {
+    "eng-package-utils": "file:eng/tools/eng-package-utils",
```

Comment:
@praveenkuttappan This file is not under the container registry folder. Any reason this got included in this PR?

Reply:
Not sure but this is already included in main branch. I merged main and pushed an update